flagging_site/data/database.py

```diff
@@ -71,3 +68,5 @@ def update_database():
     from .model import all_models
     model_outs = all_models(df)
     model_outs.to_sql('model_outputs', **options)
+
+    execute_sql_from_file('define_boathouse.sql')
```

Comment:
This shouldn't be in `update_database`. This is part of the creation of the database; just needs to be written once and that's it.

Reply:
Put in init_db()